boards/common/msba2/Makefile.include

```diff
@@ -4,7 +4,11 @@ export CPU = lpc2387
 FLASHER = lpc2k_pgm
 
 # configure serial interface
-export TERMFLAGS += -tg -p "$(PORT)"
+PORT_LINUX ?= /dev/ttyUSB0
+# This does not make a lot of sense, but it has the same value as the previous code
+PORT_DARWIN ?= /dev/ttyUSB0
```

Comment:
Tested on `milhouse` (`Darwin milhouse.imp.fu-berlin.de 14.5.0 Darwin Kernel Version 14.5.0: Sun Jun  4 21:40:08 PDT 2017; root:xnu-2782.70.3~1/RELEASE_X86_64 x86_64`): the port is `/dev/tty.usbserial-ARM`.

Reply:
Does this seem good then?
```suggestion
PORT_DARWIN ?= /dev/tty.usbserial-ARM
```

Reply:
Yes :+1:

Reply:
You may squash this immediately in. Then we can merge this.